single.php

```diff
@@ -14,6 +14,7 @@
     $local_lang = '_'.$languages[1];
 } else {
     $lang = '';
+    echo "ttit";
```

Comment:
@acorbi It was my first testing when I start using the Fork account. I will take a look at that again